app/assets/javascripts/habits.js

```diff
@@ -17,6 +21,39 @@ $(document).ready(function() {
 
             $(complete_div).toggle();
             $(incomplete_div).fadeToggle();
+
+            $.getJSON(x, function(datax) {
+                      $(function () {
+                          $('#container2').highcharts({
+                              chart: {
+                                  type: 'bar',
+                                  style: {
+                                        fontFamily: 'Quicksand'
+                                    },
+                              },
+                              title: {
+                                  text: 'Daily Points'
+                              },
+                              xAxis: {
+                                  categories: [ 'Sunday','Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday', 'Saturday']
+                              },
+                              yAxis: {
+                                  max: 100,
+                                  title: {
+                                      text: 'Points Earned'
+                                  }
+                              },
+                              series: [{
+                                  animation: false,
+                                  showInLegend: false,
+                                  name: "Points",
+                                  data: [ datax["Sunday"], datax["Monday"], datax["Tuesday"], datax["Wednesday"], datax["Thursday"], datax["Friday"], datax["Saturday"]],
```

Comment:
Line is too long.<br>['Sunday'] is better written in dot notation.<br>['Monday'] is better written in dot notation.<br>['Tuesday'] is better written in dot notation.<br>['Wednesday'] is better written in dot notation.<br>['Thursday'] is better written in dot notation.<br>['Friday'] is better written in dot notation.<br>['Saturday'] is better written in dot notation.